src/main/java/com/nylas/AvailabilityQuery.java

```diff
@@ -59,8 +65,8 @@ public Q openHours(List<OpenHours> openHours) {
 		return self();
 	}
 
-	public Q calendars(List<Map<String, List<String>>> calendars) {
-		this.calendars = calendars;
+	public Q calendars(FreeBusyCalendars... calendars) {
```

Comment:
will this break current users? should the old calendars signature be left in there and deprecated?

Reply:
Yes and no. This was launched prematurely before the feature launched on the API, and the design on the API was clarified that it must be a mapping of an account ID to calendar IDs, which a model serves better than just telling the user it's a generic List of Maps. Keeping it as is runs a strong risk that the request might fail unless the user goes to the API reference guide and finds the structure.

But _technically_ this is a breaking change, so you do have a point regardless. I will add it back, deprecate it, and add a useful javadoc to indicate the proper structure of what's expected.